cmd/checks.go

```diff
@@ -272,12 +286,12 @@ func dirContainsRegex(dirName, expressionString string) (bool, error) {
 // fileEquals calculates the SHA1 sum of a file and compares it
 // with the hash provided in the check
 func fileEquals(fileName, fileHash string) (bool, error) {
-	fileContent, err := readFile(fileName)
+	fileContent, err := ioutil.ReadFile(fileName)
```

Comment:
Any reason to not use our readFile wrapper?

Reply:
Hmmm let me change that and see what the CI says